### PR TITLE
Fix Docker build issue [RDY]

### PR DESCRIPTION
**Pull request by aravindprasads**
1) Fix for issue - https://github.com/radcli/radcli/issues/46 
 
Description:
"yum install -y yum-plugin-ovl" was needed in Docker file for the build to pass.

from MAN page:
Opening a file on OverlayFS in read-only mode causes the file from
       lower layer to be opened, then later on, if the same file is opened
       in write mode, a copy-up into the upper    layer    takes    place,
       resulting into a new file being opened.
       Since yum(8) needs to open the RPMdb first read-only, and then
       also with write access, we need to copy-up the files beforehand to
       make sure that the access is consistent.

2) Cleanup of run-server.sh file to display the Server IP and command to run the tests

### CODE REVIEW

**Comment by aravindprasads** [comment]
Hi @nmav 

Kindly review the changes and let me know your views.

File: tests/docker/Dockerfile-radius [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM fedora:21
 
+RUN yum install -y yum-plugin-ovl
```
Comment (nmav) [inline tests/docker/Dockerfile-radius:3]: what is the problem this change is solving?

File: tests/run-server.sh [hunk 2]
```diff
@@ -100,6 +100,11 @@ if test -z "$IP";then
 	echo "Detected IP is null!"
 	stop
 fi
-echo "$IP"
+
+serv_ip=$( echo $IP | cut -d' ' -f1 )
```
Comment (nmav) [inline tests/run-server.sh:104]: is this related?

File: tests/docker/Dockerfile-radius (hunk 1 above)
Comment (aravindprasads) [reply tests/docker/Dockerfile-radius:3]: This fixes the issue - https://github.com/radcli/radcli/issues/46 

I face the above issue each time. If the fix is not correct, kindly assist on the same.

File: tests/run-server.sh (hunk 2 above)
Comment (aravindprasads) [reply tests/run-server.sh:104]: Currently, we do not clearly show the Server IP address after starting the docker. 

Hence, made the changes to show the Server IP and command to start the tests clearly.

Let me know your views.

File: tests/docker/Dockerfile-radius (hunk 1 above)
Comment (nmav) [reply tests/docker/Dockerfile-radius:3]: I am a bit hesitant here because I cannot really reproduce it and the ovl plugin doesn't really make sense to me. Have you tried updating to a different fedora release (e.g. 27 or 28) fixes the issue? If yes, we can update that part instead. Overall I was thinking that maybe we get rid of docker dependency here and use namespaces directly with a script such as:
https://gitlab.com/openconnect/ocserv/blob/master/tests/ns.sh but never got into that. Anyway for a quick fix most likely changing the fedora release would work.

File: tests/run-server.sh (hunk 2 above)
Comment (nmav) [reply tests/run-server.sh:104]: ok

File: tests/docker/Dockerfile-radius (hunk 1 above)
Comment (nmav) [reply tests/docker/Dockerfile-radius:3]: Ok

Comment (aravindprasads) [reply tests/docker/Dockerfile-radius:3]: Hi @nmav 

I will check and get back.
